src/components v2/Atoms/Forms/InputText/InputText.vue

```diff
@@ -0,0 +1,169 @@
+<template>
+  <div class="ph-input-text" :class="inFocus && 'ph-input-text-focus'">
+    <div v-if="iconLeft" class="ph-input-text-icon ph-input-text-icon-left">
+      <p-icon :name="iconLeft" type="lg" />
+    </div>
+    <input
+      :id="id"
+      class="ph-input-text-input"
+      :class="{
+        'ph-input-text-input-icon-left': iconLeft,
+        'ph-input-text-icon-right': iconRight,
+        'ph-input-text-icon-center': centered,
+        'input-error': error,
+      }"
+      :placeholder="placeholder"
+      :type="!$attrs.type ? 'text' : number === true ? 'number' : $attrs.type"
+      :disabled="$attrs.disabled"
```

Comment:
**issue:** we should not reference $attrs when we know we're going to need the values.

Let's add disabled into the props.